.github/workflows/pr-auto-fix.yaml

```diff
@@ -68,12 +68,11 @@ jobs:
       - name: Install Python Interpreter
         uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
         with:
-          python-version: 3.8
+          python-version: 3.11
       - name: Install Python Packages
         run: |
           pip install pyyaml mako virtualenv absl-py
-          sudo apt-get update
-          sudo apt-get install python3-dev
+          pip list
```

Comment:
Is this needed?

Reply:
It is not _needed_, but is very useful since we don't pin the dependencies here.